src/extension/_locales/en/messages.json

```
@@ -657,5 +657,41 @@
     },
     "back": {
       "message": "Back"
+    }, 
+    "reload_success": {
+      "message": "Preview successfully updated, reloading..."
+    }, 
+    "preview_success": {
+      "message": "Preview successful, opening preview..."
+    }, 
+    "publish_success": {
+      "message": "Publish successful, opening publish..."
```

Publish is an action, not an environment, so "opening publish" sounds odd to me. How about:
```suggestion
      "message": "Publication successful, opening $1..."
```
`$1` is either `this.i18n('live')` or `this.i18n('production')` based on `this.siteStore.host`